ersilia_precalc_poc/read.py

```diff
@@ -0,0 +1,30 @@
+import pandera as pa
+from pandera.typing import DataFrame
+
+from ersilia_precalc_poc.models import Prediction, PredictionSchema
+
+# def read_predictions_from_s3(model_id: str, s3_config:) -> DataFrame[PredictionSchema]
+
+
+@pa.check_types
+def get_predictions_from_dataframe(model_id: str, prediction_df: DataFrame[PredictionSchema]) -> list[Prediction]:
```

Comment:
...then we specify the schema we created as the type hint for the function argument, and add the `pa.check_types` decorator so that inputs are validated on initialisation